lib/ProductOpener/Display.pm

```diff
@@ -2413,7 +2419,7 @@ HTML
 		
 		}
 		else {
-			$html .= "<p>$html_count" . lang("sep") . ":</p>";
+			$html .= "<p>$html_count " . lang("sep") . ":</p>";
```

Comment:
Actually the same as in [master](https://github.com/openfoodfacts/openfoodfacts-server/blame/b1a3ad30e03d2b900dea393bbbb952d5542c0175/lib/ProductOpener/Display.pm#L2451).